config/config_test.go

```diff
@@ -33,6 +32,7 @@ func TestConfig(t *testing.T) {
 				So(cfg.DefaultLimit, ShouldEqual, 10)
 				So(cfg.DefaultMaximumLimit, ShouldEqual, 50)
 				So(cfg.DefaultMaximumSearchResults, ShouldEqual, 500)
+				So(cfg.SiteDomain, ShouldEqual, "localhost")
 			})
```

Comment:
Should there be tests for the other new config variables? `Debug` and `SupportedLanguages`?

Reply:
👍